lib/big_query/timestamp_rollups/daily.ex

```diff
@@ -0,0 +1,39 @@
+defmodule BigQuery.TimestampRollups.Daily do
+  @behaviour BigQuery.TimestampRollup
+
+  def name, do: "DAY"
+
+  def rollup, do: "TIMESTAMP_TRUNC(timestamp, DAY)"
+
+  def is_a?(param), do: Enum.member?(["1d", "DAY"], param)
+
+  def floor(time) do
+    Timex.beginning_of_day(time)
+  end
+
+  def ceiling(time) do
+    if Timex.compare(floor(time), time) == 0 do
+      time
+    else
+      Timex.end_of_day(time) |> Timex.shift(microseconds: 1)
```

Comment:
porquoi the microseconds shift on here + not the hourly / 15min?

Reply:
Those `Timex.end_of_` functions return 1-millisecond short of the next day/week/month.  So I have to shift them slightly to get back to 00:00:00.  As to why not hourly/15min ... Timex doesn't supply helpers for those.  Otherwise yeah, those would have been easier/cleaner to use!